spec.html

```diff
@@ -28653,6 +28655,27 @@ <h1>Runtime Semantics: GetSubstitution( _matched_, _str_, _position_, _captures_
                   The _nn_<sup>th</sup> element of _captures_, where _nn_ is a two-digit decimal number in the range 01 to 99. If _nn_&le;_m_ and the _nn_<sup>th</sup> element of _captures_ is *undefined*, use the empty String instead. If _nn_ is 00 or _nn_&gt;_m_, no replacement is done.
                 </td>
               </tr>
+              <tr>
+                <td>
+                  0x0024, 0x003C
+                </td>
+                <td>
+                  `$&lt;`
+                </td>
+                <td>
+                  <emu-alg>
+                    1. If _namedCaptures_ is *undefined*, the replacement text is the String `"$&lt;"`.
+                    1. Otherwise,
+                      1. Scan until the next `>`.
```

Comment:
Probably could use the new single unicode codepoint convention here (U+XXXX CHAR NAME)